### ISSUE TITLE
bug: axe DevTools reports several items as "not announced by screen reader" 

### ISSUE THREAD

**Issue by kensodemann** [ionic-team/ionic-framework#23511]
# Bug Report

**Ionic version:**

[ ] **4.x**
[x] **5.x**
[ ] **6.x**

**Current behavior:**

Components such as `ion-select`, `ion-toggle`, and `ion-radio` report with the following error:
https://dequeuniversity.com/rules/axe/4.2/nested-interactive?application=AxeChrome

However, in my testing, they do appear to _actually_ be read by the screen reader when you do a "read from top" action (though I only tested on my iPhone to this point).

**Expected behavior:**

Ideally, no AXE errors would be displayed if possible.

**Steps to reproduce:**
A repro case can be found here: https://codesandbox.io/s/ionic-axe-42-failures-1y1pq

**Comment by liamdebeasi** [comment]
Thanks for the report. This should be resolved in Ionic 7 using the new form syntax. The Ionic 7 migration guide can be found here: https://ionicframework.com/docs/v7/updating/7-0
